keras_cv/layers/preprocessing/fourier_mix.py

```diff
@@ -182,9 +165,7 @@ def _fourier_mix(self, images):
 
         # binarise masks utilizing mapped calls
         masks = tf.map_fn(
```

Comment:
@quantumalaviya is it possible to use `tf.vectorized_map` instead?

Reply:
I was just wondering this. I am running the experiment rn!

Reply:
Much better.  Performance comes second to correctness.